src/invenio-files-js/controllers/InvenioFilesCtrl.js

```diff
@@ -388,6 +392,10 @@ function InvenioFilesCtrl($rootScope, $scope, $q, $timeout,
   vm.remove = removeFile;
   // Start the upload
   vm.upload = upload;
+  // Push to upload queue
+  vm.pushToQueue = function(x) {
+    return Uploader.pushToQueue(x);
+  };
```

Comment:
can we move it to a separated function as the ``vm.upload``? It is better to avoid anonymous functions :)